css/templates/e-commerce/page.css

```diff
@@ -590,6 +591,10 @@ amp-carousel {
   line-height: 1.6;
 }
 
+.commerce-blog-sidebar .ampstart-social-follow li:first-child a {
```

Comment:
This is quite a long selector but I couldn't find an easy way to do this apart from adding a class via json, but it wouldnt adhere to the design doc naming style, so would therefore need to change all templates inheriting this component.